### PR TITLE
Responsive styling for dandiset list

### LINKED ISSUE #636: css fix for user experience
would it be possible to fix the css/vuetify controls such that the top row and each individual item in the list fit the view.

image

### PR DESCRIPTION
**Pull request by jtomeck**
Closes #636

### CODE REVIEW

**Comment by jtomeck** [comment]
I'm unsure why `test_and_deploy` is failing, as the watcher isn't displaying any errors.

**Comment by satra** [comment]
@jtomeck - i just merged master in case it makes a difference.

File: web/src/components/DandisetList.vue [hunk 1]
```diff
@@ -144,3 +139,20 @@ export default defineComponent({
...
+}
+.v-list-item__title {
+  white-space: normal;
+}
+
+.v-list-item {
+  border-bottom: 1px solid #eee;
+}
+
+.v-list--three-line .v-list-item .v-list-item__content, .v-list-item--three-line .v-list-item__content {
```
Comment (satra) [inline web/src/components/DandisetList.vue:155]: appears to be a linting error on this line being too long.

Comment (jtomeck) [reply web/src/components/DandisetList.vue]: Oh thanks, I'm not sure why this didn't throw an error in my `yarn run serve` but it is resolved now

**Comment by satra** [comment]
@jtomeck - nice - the changes works for me.

**Review by satra — APPROVED** [review]
LGTM.

**Pull request merged**